core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -48,6 +48,7 @@ angular.module('oppia').directive('topNavigationBar', [
             SiteAnalyticsService, NavigationService, WindowDimensionsService,
             DebouncerService, DeviceInfoService, LOGOUT_URL) {
           var ctrl = this;
+          this.$onInit = function() {
           ctrl.isModerator = null;
```

Comment:
Fix indentation please, here and elsewhere.

Reply:
done Thanks